apps/web/playwright/webhook.e2e.ts

```diff
@@ -427,7 +427,7 @@ test.describe("BOOKING_RESCHEDULED", async () => {
 
     await page.locator('[data-testid="confirm-reschedule-button"]').click();
 
-    await expect(page).toHaveURL(/.*booking/);
+    await expect(page.getByTestId("success-page")).toBeVisible();
```

Comment:
What is the possible reason for `toHaveUrl` to be not working?

Reply:
I am not sure why it happened, but the issue was that sometimes `.toHaveURL(/.*booking/)` was already satisfied but the booking was not yet in the db